spec/configs/stanford_grants_to_sparql_statements_spec.rb

```diff
@@ -13,7 +13,8 @@
     "projectStartDate": "2017-07-01T00:00:00.000-07:00",
     "projectEndDate": "2018-11-01T00:00:00.000-07:00",
     "directSponsorName": "The William and Flora Hewlett Foundation",
-    "piEmployeeId": "12345678"
+    "piFullName": "Giarlo, Michael J.",
```

Comment:
So does this mean that we need more sophisticated name parsing to handle the middle initial?

Reply:
@justinlittman I was hoping the name variant work you did would take care of this. Ha! No?

Reply:
No. It does variations on names given a correct first, middle, last. It will optimize resolving, but does not help when creating new people.

Reply:
What we do here depends on @peetucket's answer above. Let's discuss after stand-up today and I'll spruce up this PR shortly thereafter. Thanks for the thoughtful review, @justinlittman!